js/private/test/image/BUILD.bazel

```diff
@@ -69,3 +71,21 @@ assert_js_image_layer_listings(
     name = "custom_owner_test",
     js_image_layer = ":custom_owner",
 )
+
```

Comment:
Can we see an example of excluding things? Similar to my use case wanting to exclude `@types`...

Reply:
I'll do that as a follow-up.